scripts/airdrop.js

```diff
@@ -0,0 +1,54 @@
+const fs = require("fs").promises
+
+const { signAndSend, transactionExistsOnSafeServer } = require("./utils/gnosis_safe_server_interactions")(web3, artifacts)
+const { buildTransferDataFromList } = require("./utils/trading_strategy_helpers")(web3, artifacts)
+const { promptUser } = require("./utils/user_interface_helpers")
+const { default_yargs } = require("./utils/default_yargs")
+const argv = default_yargs
+  .option("masterSafe", {
+    type: "string",
+    describe: "Address of Gnosis Safe transfering funds",
+    demandOption: true,
+  })
+  .option("transferFile", {
+    type: "string",
+    describe: "file name (and path) to the list transfers",
+    demandOption: true,
+  })
+  .option("verify", {
+    type: "boolean",
+    default: false,
+    describe: "Do not actually send transactions, just simulate their submission",
+  })
+  .option("useWei", {
+    type: "boolean",
+    describe: "boolen flag used to indicate if the amounts provided are already in atoms",
+    demandOption: false,
+  }).argv
+
+module.exports = async (callback) => {
+  try {
+    const GnosisSafe = artifacts.require("GnosisSafe")
+    const masterSafe = await GnosisSafe.at(argv.masterSafe)
+
+    const transfers = JSON.parse(await fs.readFile(argv.transferFile, "utf8"))
+
+    console.log("Preparing transaction data...")
+    const transaction = await buildTransferDataFromList(masterSafe.address, transfers, argv.useWei, false, true)
```

Comment:
ubernit: This comment doesn't belong to this PR. But just want to take the opportunity to mention something, since I see it in many parts of the codebase. I believe this way of passing parameters is considered a bad practice.

The reason is that, as a reader/reviewer, I can guess what the first 3 parameters are, but for the last 2 I would need to jump to buildTransferDataFromList implementation:
`buildTransferDataFromList(masterSafe.address, transfers, argv.useWei, false, true)` 

Further more, if I by mistake invert the order `false, true` (instead of `true`, `false``), cause I though the order was different.

For any non trivial function signature (more than one param), we could use named parameters.
```js
buildTransferDataFromList({ 
  masterAddress: masterSafe.address, 
 transferList: transfers, 
 useWei: argv.useWei, 
 unsafe: false,
 debug: true
})
```

Actually, it would allow to define default values for `unsafe` and `debug` so the call is simpler.

Reply:
In hindsight, I'd like to have most of the functions of this projects handled like that.

Reply:
I agree completely about this, but would like to save these changes for a separate PR (i.e. refactoring).

Reply:
> In hindsight, I'd like to have most of the functions of this projects handled like that.

Once this becomes a typescript project we wont need to do any of this. Essentially what is being proposed is like a hack in JS to make unsafe code ever so slightly safer than it is. If we are going to start doing this, we should just migrate to typescript where we aren't trying to trick our IDE.

Reply:
Even for typescript projects is a good pattern.

Yes, no need to address it. It's just someone I wanted to mention, since I haven't been reading that many PRs from this repo, and I just thought it was good to mention it.